app.js

```diff
@@ -1499,6 +1499,162 @@ app.delete("/room/:roomId", function (req, res) {
     });
 });
 
+
+//suggest meeting times given list of participants
+app.post("/api/meetingSuggestion", function (req, res) {
+
+    //currently only look 3 days ahead, and assume 7AM - 5PM workday. only search on weekdays
+
+    if(!req.body.participants) {
+      res.statusCode = 400;
+      console.log();
+      res.json({
+        "requestURL":  "/room",
+        "action": "post",
+        "status": 400,
+        "message": "Bad Request",
+        "timestamp": new Date()
+      });
+    }
+
+    else {
+
+      var participants = req.body.participants || null;
+
+      //using ADDDATE(CURDATE(), 4) just to limit the amount of meetings to search through later
+      var getMeetingSql = "SELECT start_datetime, end_datetime FROM ebdb.meeting WHERE " //Where end_datetime < ADDDATE(CURDATE(), 4) AND"
```

Comment:
Did you remove the filter on getting meetings because you were having trouble getting it working?  I would suggest we filter out any meetings that end in the past or start beyond our look-up datetime - which would be specified by a variable we default above.

Reply:
Yea, I had trouble getting it work but I'll try again